src/lib/ngsi/ContextElementResponseVector.h

```diff
@@ -44,14 +44,16 @@ typedef struct ContextElementResponseVector
   std::string              render(ConnectionInfo*     ciP,
                                   RequestType         requestType,
                                   const std::string&  indent,
-                                  bool                comma = false);
+                                  bool                comma               = false,
+                                  bool                omitAttributeValues = false);
 
   void                     present(const std::string& indent);
   void                     push_back(ContextElementResponse* item);
   unsigned int             size(void);
   ContextElementResponse*  get(unsigned int ix);
-  ContextElementResponse*  lookup(EntityId* eP);
+  ContextElementResponse*  lookup(EntityId* eP, HttpStatusCode code = SccNone);
```

Comment:
I understand that after this PR, the SccNotFound variable (in HttpStatusCode.h I guess) is no longer needed. Is has been removed from that file?